dbsync-central/release_stuff/scripts/debezium_offset_bkp.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+# script for backup the debezium off-set file
+#
+
+# Set EIP environment.
+HOME_DIR=/home/eip
+SCRIPTS_DIR=$HOME_DIR/scripts
+BKPS_HOME=/home/eip/shared/bkps
+DEBEZIUM_HOME=/home/eip/shared/.debezium
+DEBEZIUM_OFFSET_FILE=$DEBEZIUM_HOME/offsets.txt
+DEBEZIUM_HISTORY_FILE=$DEBEZIUM_HOME/dbhistory.txt
+timestamp=`date +%Y-%m-%d_%H-%M-%S`
+DEBEZIUM_OFFSET_FILE_BKP="offsets.txt$timestamp"
```

Comment:
All back ups should be stored on the host and not inside the containers

Reply:
This is a volume dir located in host